server/index.js

```diff
@@ -37,8 +37,21 @@ app.get('/connection', protectRoute, (req, res) => {
         return;
       }
       if (socket) {
-        socket.emit('token', createToken(foundUser.pet_name || 'Pet'));
-        res.json({ token: createToken(foundUser.name) });
+        const room = foundUser.room || 'Room 1';
+        socket.emit('token', {
+          token: createToken({
+            identity: foundUser.pet_name || 'Pet',
```

Comment:
you're sending arguments as an object but createToken is not an object. wrap {} in createToken in twilio.js if you want to do it this way.